.rubocop.yml

```diff
@@ -0,0 +1,19 @@
+Metrics/MethodLength:
```

Comment:
We have a standard Rubocop config I've been using in all our cookbooks.  You can find it here: https://github.com/chef-cookbooks/keepalived/blob/master/.rubocop.yml